src/reaktoro_pse/tutorials/basic_reaktoro_block_interaction.ipynb

```diff
@@ -606,40 +612,8 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "2024-10-28 19:04:31 [INFO] idaes.reaktoro_pse.reaktoro_block: -----Displaying information for property block ------\n",
-      "2024-10-28 19:04:31 [INFO] idaes.reaktoro_pse.reaktoro_block: +-----------------+------------+------+\n",
-      "| Property        |      Value | Unit |\n",
-      "+-----------------+------------+------+\n",
-      "| Temperature     |   293.0000 |    K |\n",
-      "| Pressure        |     1.0000 |  bar |\n",
-      "| Charge:         | 5.5771e-02 |  mol |\n",
-      "| Element Amount: |            |      |\n",
-      "| :: H            | 1.1101e+02 |  mol |\n",
-      "| :: C            | 2.3329e-03 |  mol |\n",
-      "| :: O            | 5.5623e+01 |  mol |\n",
-      "| :: Na           | 4.5917e-01 |  mol |\n",
-      "| :: Mg           | 5.1926e-02 |  mol |\n",
-      "| :: S            | 2.7575e-02 |  mol |\n",
-      "| :: Cl           | 4.7950e-01 |  mol |\n",
-      "| :: K            | 9.7192e-03 |  mol |\n",
-      "| :: Ca           | 9.9803e-03 |  mol |\n",
-      "| Species Amount: |            |      |\n",
-      "| :: H+           | 3.8058e-08 |  mol |\n",
-      "| :: H2O          | 5.5506e+01 |  mol |\n",
-      "| :: CO3-2        | 1.9905e-05 |  mol |\n",
-      "| :: CO2          | 9.1011e-05 |  mol |\n",
-      "| :: Ca+2         | 9.9803e-03 |  mol |\n",
-      "| :: Cl-          | 4.7950e-01 |  mol |\n",
-      "| :: HCO3-        | 2.2039e-03 |  mol |\n",
-      "| :: SO4-2        | 2.7575e-02 |  mol |\n",
-      "| :: HSO4-        | 9.0296e-09 |  mol |\n",
-      "| :: Mg+2         | 5.1907e-02 |  mol |\n",
-      "| :: K+           | 9.7192e-03 |  mol |\n",
-      "| :: MgCO3        | 1.8022e-05 |  mol |\n",
-      "| :: MgOH+        | 4.1372e-07 |  mol |\n",
-      "| :: Na+          | 4.5917e-01 |  mol |\n",
-      "| :: OH-          | 4.0668e-07 |  mol |\n",
-      "+-----------------+------------+------+\n"
+      "2025-03-25 14:05:58 [INFO] idaes.reaktoro_pse.reaktoro_block: -----Displaying information for property block ------\n",
+      "<reaktoro_pse.core.reaktoro_state.ReaktoroState object at 0x00000266F5B13E90>\n"
```

Comment:
It looks like something changed in the cell output, i.e. the internal `repr` for the `ReaktoroState` instance is shown rather than the human-readable table.

Reply:
Good catch, I pushed to fix all of the issues you noted on J notebooks I think should be good

Reply:
Its becouse I did not push the re-ran jupyter output